src/MediaUpload.js

```diff
@@ -69,7 +93,15 @@ class MediaUpload extends Component {
         let fileIndex = 0;
         let fileUploads = this.state.filesList.map((isUploaded) => {
             fileIndex++;
-            return (<FileUpload key={fileIndex} fileName={fileIndex}/>);
+            let dbxOptions;
+            if (this.props.uploadName === "Images") {
```

Comment:
Pass file type to dbx choose function and do this logic in the dropbox driver

Reply:
done